frontend/src/components/search/SearchFilterEligibility.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import {
+  FilterOption,
+  SearchFilterAccordion,
+} from "src/components/search/SearchFilterAccordion/SearchFilterAccordion";
+
+export interface SearchFilterEligibilityProps {
+  initialQueryParams: Set<string>;
+  formRef: React.RefObject<HTMLFormElement>;
+}
+
+export default function SearchFilterEligibility({
```

Comment:
Note that I did modify the naming on a few of these to be shorter as the existing values were a bit long to have as a constant.

You can see the original values on https://grants.gov/system-to-system/grantor-system-to-system/schemas/grants-funding-synopsis#EligibleApplicantTypes

For example: `Nonprofits having a 501(c)(3) status with the IRS, other than institutions of higher education` became `nonprofits_non_higher_education_with_501c3`. The former might be better for the label on the front-end.

Reply:
@andycochran I'm going to use the longer one for the label here per @chouinar's suggestion:

![image](https://github.com/HHS/simpler-grants-gov/assets/93001277/8e97adef-a8e1-4406-b2bd-86659b327cdf)